AdyenCard/Form/FormCardSecurityCodeItemView.swift

```diff
@@ -15,7 +15,7 @@ internal final class FormCardSecurityCodeItemView: FormTextItemView<FormCardSecu
         accessory = .customView(cardHintView)
         observe(item.$selectedCard) { [weak self] cardsType in
             let number = cardsType == CardType.americanExpress ? "4" : "3"
-            let localization = ADYLocalizedString("adyen.card.cvcItem.placeholder.digits", item.localizationParameters)
+            let localization = localizedString(.cardCvcItemPlaceholderDigits, item.localizationParameters)
```

Comment:
We can use here `localizedString(. cardCvcItemPlaceholderDigits, item.localizationParameters, number)`

Reply:
👍